app/assets/javascripts/listeners.js

```diff
@@ -1,66 +1,74 @@
-		{id: "signin_for_request_options", category: "search-results"},
-		{id: "request", category: "search-results"},
-		{id: "available_button", category: "search-results"},
-		{id: "navbar_everything", category: "navigation"},
-		{id: "navbar_books", category: "navigation"},
-		{id: "navbar_articles", category: "navigation"},
-		{id: "navbar_journals", category: "navigation"},
-		{id: "navbar_more", category: "navigation"},
-		{id: "request_options", category: "search-results"},
-		{id: "request-btn-0", category: "search-results"}
-	];
+		const tracks = [
+			{id: "logo-navbar", category: "navigation"},
+			{id: "my-account", category: "navigation"},
+			{id: "login", category: "navigation"},
+			{id: "logout", category: "navigation"},
+			{id: "bookmarks_nav", category: "navigation"},
+			{id: "navbar_everything", category: "navigation"},
+			{id: "navbar_more", category: "navigation"},
+			{id: "navbar_articles", category: "navigation"},
+			{id: "navbar_databases", category: "navigation"},
+			{id: "navbar_journals", category: "navigation"},
+			{id: "navbar_website", category: "navigation"},
+			{id: "startOverLink", category: "search-results"},
+			{id: "start_over", category: "search-results"},
+			{id: "back_to_search", category: "search-results"},
+			{id: "advanced-search-submit", category: "search-results"},
+			{id: "articles_advanced_search", category: "search-results"},
+			{id: "databases_advanced_search", category: "search-results"},
+			{id: "journals_advanced_search", category: "search-results"},
+			{id: "catalog_advanced_search", category: "search-results"},
+			{id: "search", category: "search-results"},
+			{id: "bento_books_and_media", category: "bento-results"},
+			{id: "bento_articles", category: "bento-results"},
+			{id: "bento_databases", category: "bento-results"},
+			{id: "bento_website", category: "bento-results"},
+			{id: "bento_journals", category: "bento-results"},
+			{id: "bento_resource_types", category: "bento-results"},
+			{id: "emailLink", category: "record-page"},
+			{id: "smsLink", category: "record-page"},
+			{id: "refworksLink", category: "record-page"},
+			{id: "citationLink", category: "record-page"},
+			{id: "risLink", category: "record-page"},
+			{id: "bookmark_button", category: "record-page"},
+			{class: "bento-online", category: "bento-results"},
+			{class: "online-btn", category: "search-results"},
+			{class: "request-button", category: "search-results"},
+			{class: "availability-btn", category: "search-results"},
+		];
 
-  function handleEventClicks(event) {
-    if (typeof ga != "undefined") {
-      ga('send', 'event', {
-        eventCategory: 'Click Event',
-        eventAction: 'click',
-        eventLabel: event,
-        forceSSL: true,
-        anonymizeIp: true
-      });
-    }
-  }
+	  const handleEventClicks = (label, category) => {
+	    if (typeof ga != "undefined") {
+	      ga("send", "event", {
+	        eventCategory: "category",
```

Comment:
This needs to be `category`
```suggestion
	        eventCategory: category,
```

Reply:
We were waiting for the okay from Emily before using categories.